pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx

```diff
@@ -35,12 +28,10 @@ export const PokemonDisplay = (props: displayProps) => {
     (url: string) => {
       if (url && url.length > 0) {
         getPokemonData(url)
```

Comment:
Something that I want to do in another submission is convert this to use something other than a URL to get the data out of the service.

Unless, of course, we use the URL as the repository key. While possible, I don't really like it, but it does solve some of our issues with retrieval.

Reply:
What do you mean? You want to use the Name/Url interface you created?